prep-html-syntax/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <body>
+      <h1 style="color: pink;">My favorite Star Wars character is no one because I've never seen them lol</h1>
```

Comment:
Hi Avallon, be sure to close your `<head>` before opening your `<body>`.

Reply:
![image](https://user-images.githubusercontent.com/96964291/149067069-bb4d8a61-d892-4d82-8b54-519955d57970.png)

![image](https://user-images.githubusercontent.com/96964291/149067090-8c0eab92-338b-4a63-a184-8cd0e4f29833.png)

I believe I have corrected the error, thank you for the feedback!